SPECS/libacvp/libacvp.spec

```diff
@@ -0,0 +1,61 @@
+Summary:        A library that implements the client-side of the ACVP protocol
+Name:           libacvp
+Version:        1.2.0
+Release:        1%{?dist}
+License:        Apache License 2.0
```

Comment:
Please use the short name from [Fedora's list](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#License_of_Fedora_SPEC_Files) as per our spec guidelines. In this case it's probably "ASL 2.0".